src/functionalTest/java/uk/gov/hmcts/reform/locationrefdata/cameltest/LrdApplicationExceptionAndAuditTest.java

```diff
@@ -110,29 +109,6 @@ void testTaskletFailure() throws Exception {
         validateLrdServiceFileAudit(jdbcTemplate, auditSchedulerQuery, "Failure", UPLOAD_ORG_SERVICE_FILE_NAME);
     }
 
-    private void testInsertion() throws Exception {
```

Comment:
Is there a reason behind the removal of this method? The PR doesn't show any other changes w.r.to uploading the files and launching the job in this test class.

Reply:
this is an existing unused method. so removed it

Reply:
Also, I have used the existing exceptionTest written to validate the rowid